partner_aging/wizard/res_partner_aging_customer.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2022 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, tools
+
+
+class ResPartnerAgingCustomer(models.Model):
+    _name = "res.partner.aging.customer"
+    _description = "Res Partner Aging Customer"
+    _auto = False
+    _order = "partner_id"
+
+    partner_id = fields.Many2one("res.partner", "Partner", readonly=True)
+    avg_days_overdue = fields.Integer(readonly=True)
+    date = fields.Date(readonly=True)
+    date_due = fields.Date("Due Date", readonly=True)
+    inv_date_due = fields.Date("Invoice Date", readonly=True)
+    total = fields.Float(readonly=True)
+    not_due = fields.Float("Not Due Yet", readonly=True)
+    days_due_01to30 = fields.Float("1/30", readonly=True)
+    days_due_31to60 = fields.Float("31/60", readonly=True)
+    days_due_61to90 = fields.Float("61/90", readonly=True)
+    days_due_91to120 = fields.Float("91/120", readonly=True)
+    days_due_121togr = fields.Float("+121", readonly=True)
+    max_days_overdue = fields.Integer("Days Outstanding", readonly=True)
+    invoice_ref = fields.Char("Our Invoice", size=25, readonly=True)
+    invoice_id = fields.Many2one("account.move", "Invoice", readonly=True)
+    salesman = fields.Many2one("res.users", "Sales Rep", readonly=True)
```

Comment:
Remove readonly attributes and put it in view please

Reply:
I'm not sure about that. Can you please explain why?

Reply:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#:~:text=If%20a%20field%20is%20marked%20as%20readonly%3DTrue%20on%20the%20model%2C%20it%20won%27t%20be%20possible%20to%20import%20it%20through%20the%20Odoo%20import%20tool%2C%20so%20avoid%20it%20as%20possible%20and%20define%20the%20readonly%20attribute%20on%20the%20views%20instead.